tests/__snapshots__/files/prism-core.js.shot

```diff
@@ -278,11 +278,11 @@ var Prism = (function (_self) {
 			 *
 			 * @example
 			 *   Prism.languages['css-with-colors'] = Prism.languages.extend('css', {
-			 *     // Prism.languages.css already has a 'comment' token, so this token will overwrite CSS' 'comment' token
-			 *     // at its original position
-			 *     'comment': { ... },
-			 *     // CSS doesn't have a 'color' token, so this token will be appended
-			 *     'color': /\\\\b(?:red|green|blue)\\\\b/
+			 *       // Prism.languages.css already has a 'comment' token, so this token will overwrite CSS' 'comment' token
+			 *       // at its original position
```

Comment:
The indentation of example code is now properly detected and removed. In Prism's case, the example code wasn't indented with 2 spaces before, so nothing is removed.